extras/3_nfs.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/bash
+
+WINDOWS_IMAGE="http://10.8.120.188/Summit2019.raw"
+#SERVER_IP="192.168.111.1"
+SERVER_IP=$( ip -f inet addr show baremetal |  grep inet | awk '/inet / {print $2}' | cut -d/ -f1)
+VM_SIZE="60"
+VM_MEMORY="8192"
+
+SERVERS=$(oc get nodes -o jsonpath={.items[*].status.addresses[?\(@.type==\"InternalIP\"\)].address} | xargs)
+for server in $SERVERS; do
+   ssh core@$server "sudo bash -c 'curl http://karmatron.mooo.com/cnv-bridge > /var/lib/cni/bin/cnv-bridge' ; sudo chmod u+x /var/lib/cni/bin/cnv-bridge"
```

Comment:
Can't the cnv-bridge file be hosted in this repo to eliminate an external dependency on karmatron.mooo.com?

Reply:
Also, how is this related to NFS?

Reply:
it s not really related to nfs but to the overall windows vm, so i renamed the script accordingly.
i switched the url of cnv bridge to some public gist to eliminate the dependency, and because i dont want to store a binary in the repo